src/apps/inventory/models.py

```diff
@@ -0,0 +1,19 @@
+from django.db import models
+
+
+class Category(models.Model):
+    name = models.CharField(max_length=255)
+    description = models.TextField(blank=True)
+
+    def __str__(self):
+        return self.name
+
+
+class Product(models.Model):
+    name = models.CharField(max_length=255)
+    description = models.TextField(blank=True)
+    category = models.ForeignKey('Category', on_delete=models.CASCADE)
```

Comment:
I'm not sure if we should use CASCADE here. Using CASCADE means that if we remove a category, all related products will also be removed.

Another question is how many categories a product may belong to, is it just one or more, what about product without category?

This question pertains more to our business logic rather than the technical aspect.

@egorvavilov what do you think? I think that associating a product with one or more categories will be a greater technical challenge, but it is closer to the real-world scenario.

Reply:
I confirm, Let's use PROTECT to be 100% sure user must do smth manually with Products (migrate them to other catO or just delete)